keyboards/4pplet/waffling60/rev_e_iso/info.json

```
@@ -0,0 +1,406 @@
+{
+    "manufacturer": "4pplet",
+    "keyboard_name": "waffling60 Rev E ISO HS",
+    "maintainer": "4pplet",
+    "bootloader": "stm32-dfu",
+    "diode_direction": "COL2ROW",
+    "encoder": {
+        "rotary": [
+            {"pin_a": "A2", "pin_b": "A1", "resolution": 2}
+        ]
+    },
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "key_lock": true,
+        "mousekey": false,
```

This is preferred to be left enabled.
```suggestion
        "mousekey": true,
```